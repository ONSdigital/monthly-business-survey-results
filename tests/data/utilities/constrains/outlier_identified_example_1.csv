reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value
1,202001,14,42,100,1,100
1,202001,14,43,800,0.8,640
1,202001,14,40,900,1,900
