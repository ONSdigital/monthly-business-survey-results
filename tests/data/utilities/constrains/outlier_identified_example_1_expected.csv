reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value,post_winsorised
1,202001,14,42,100,1,100.0,FALSE
1,202001,14,43,800,0.8,640.0,FALSE
1,202001,14,40,900,0.822222222,740.0,TRUE
